tests/testthat/test-carehomes-multistrain.R

```diff
@@ -1295,22 +1292,42 @@ test_that("If prob_strain is NA then Rt is NA ", {
   R <- y[index_R, , ]
   prob_strain <- y[index_prob_strain, , ]
 
-  ## all values of prob_strain after the first step should be NA
-  expect_true(all(is.na(prob_strain[, , -1L])))
+  ## set NA for prob_strain in steps 60-70
+  prob_strain[, , 60:70] <- NA
```

Comment:
especially given the lintr false positive here, how about:

```r
drop <- 60:70
expect_true(!any(is.na(prob_strain[, , drop])))
expect_true(all(is.na(prob_strain[, , drop])))
```

etc (rename `drop` as you want)